plugins/direct_events_plugin/db/services.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+defined('MOODLE_INTERNAL') || die();
+
+$services = array(
+      'directeventhandler' => array(                                                // the name of the web service
+          'functions' => array ('local_direct_event_handler'), // web service functions of this service
+          'requiredcapability' => '',                // if set, the web service user need this capability to access 
+                                                                              // any function of this service. For example: 'some/capability:specified'                 
+          'restrictedusers' => 0,                                             // if enabled, the Moodle administrator must link some user to this service
+                                                                              // into the administration
+          'enabled' => 1,                                                       // if enabled, the service can be reachable on a default installation
+          'shortname' =>  'd_handler',       // optional – but needed if restrictedusers is set so as to allow logins.
```

Comment:
I'd use a plugin namespaced convention here. If we go with calling this `fe_events_direct` than making the `shortname` something like `local_fe_events_direct_service` perhaps.